pydocstringformatter/_configuration/toml_parsing.py

```diff
@@ -41,6 +44,22 @@ def parse_toml_option(
         except KeyError:
             raise UnrecognizedOption(f"Don't recognize option {opt}") from exc
 
+    if isinstance(action, BooleanOptionalAction):
+        option_strings = action.option_strings
+
+        if opt.startswith("no") and f"--{opt[3:]}" in option_strings:
+            opposite_opt = opt[3:]
+            val, opp_val = ["false", "true"][value], ["true", "false"][value]
+            error_msg = (
+                "TOML file contains an unsupported option "
+                f"'{opt}: {val}', try using '{opposite_opt}: {opp_val}' instead"
+            )
+            raise TomlParsingError(error_msg)
+
+        index = option_strings.index(f"--{'no-' if not value else ''}{opt}")
```

Comment:
What is this looking up? The default value?

Reply:
It enforces the positive version regardless of the default value (it also does not assume that the default is the first argument).

assuming the passed value is `eggs = false`
if the option strings are:

> ['--no-eggs', '-eggs'] --> index = 0
> ['--eggs', '--no-eggs'] --> index = 1

Reply:
But you're getting the index of a string in a list to then return the string at that index again?
Can't we just do `[f"--{'no-' if not value else ''}{opt}"]`?

Reply:
you are right

```python
index = option_strings.index(f"--{'no-' if not value else ''}{opt}")
option = action.option_strings[index]
return [option]
```

should become
```
option = f"--{'no-' if not value else ''}{opt}"
return [option]
```

Reply:
You could even merge the lines and just return without the assignment in between. `return` vs. `option = ` has a shorter line length anyway

Reply:
done in https://github.com/DanielNoord/pydocstringformatter/pull/166/commits/c51e3220b66f90f68c134cf0bd777ff82e214782